.github/workflows/vmware-releases-main.yml

```diff
@@ -0,0 +1,47 @@
+name: Get the latest open-vm-tools release for main
+on:
+  schedule:
+    - cron:  '0 7 * * 5'
+
```

Comment:
If you add ` workflow_dispatch:` as a trigger, then we can manually test it right after merging.

Reply:
Good idea, added workflow_dispatch to every workflow.